lib/bulk_processor/job.rb

```diff
@@ -18,11 +18,9 @@ def perform(records, item_proccessor, handler, payload)
         end
       end
       handler_class.complete(payload, successes, failures, nil)
-    rescue => error
-      handler_class.complete(payload, successes, failures, error)
     rescue Exception => exception
       handler_class.complete(payload, successes, failures, exception)
-      raise
+      raise unless exception.is_a?(StandardError)
```

Comment:
I will add a comment in the PR built on top of this one: https://github.com/apartmentlist/bulk-processor/pull/4